language/ru.php

```diff
@@ -577,7 +577,11 @@
     'Archive to the original folder' => false,
     'Move To Blocked Folder' => false,
     'Sieve server capabilities' => false,
-    'Connection To Sieve Server Failed' => false
+    'Connection To Sieve Server Failed' => false,
+    'Missing attachment?' => 'Забыли вложение?',
+    'Did you forget to attach a file?' => 'Вы не забыли прикрепить файл?',
```

Comment:
This translation is not used anywhere

Reply:
You are right @josaphatim , I still wandering if there is no way to use trans in site.js, I would prefer to keep:
``` 
'We couldn\'t find the attachment you referred to. Please confirm if you attached it or provide the details again.' => false 
```
in lang file instead.

Reply:
Yes, let's make it possible to use trans function in JS context as well. This would entail:
- output module (in core module probably) to send existing language translations as array to the DOM (through a script tag and json output)
- having a global or window js variable with translations will make writing a trans function in JS quite easy
- use that trans everywhere you need to translate a string in JS context.

This will be useful for other js files that have static strings and need translation.

Reply:
```
window.transations 
```
and 
```
window.current_lang
```
added with trans() function available in js